calibrate.py

```diff
@@ -164,24 +173,29 @@ def recalibrate(files):
         with open('Error_Cal.txt', 'w') as file:
             file.writelines('Check calibration in %s \n' % error for error in
                             cal_error)
-    calibration_table(files, cal_headers, cal_offsets)
+    # for file in files:
+    #     if '_recal.Spe' in file:
+    #         files.remove(file)
+
+    files_updated = [os.path.splitext(files[i])
+                     [0].replace("_recal", "")
+                     for i in np.arange(len(files))]
```

Comment:
Also, no need to index here - you can just do `[os.path.splitext(f)[0] for f in files]` or so